<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Categories</class>
 <widget class="QWidget" name="Categories">
  <property name="windowTitle">
   <string>Book Categories</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #ecf0f1;
    }
    QLabel#labelTitle {
        color: #2c3e50;
        font-size: 22px;
        font-weight: bold;
        padding: 12px;
    }
    QListWidget {
        background-color: #ffffff;
        border: 1px solid #bdc3c7;
        border-radius: 6px;
        padding: 6px;
    }
    QLineEdit {
        padding: 8px;
        border: 1px solid #95a5a6;
        border-radius: 6px;
        background-color: #ffffff;
        font-size: 14px;
    }
    QPushButton {
        background-color: #3498db;
        color: white;
        padding: 8px 16px;
        border-radius: 8px;
        font-size: 14px;
        font-weight: bold;
        border: none;
    }
    QPushButton:hover {
        background-color: #2980b9;
    }
    QPushButton:pressed {
        background-color: #2471a3;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="margin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>üìÇ Book Categories</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>‚ûï Enter new category...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>‚ûï Add Category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>üóëÔ∏è Delete Category</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
